### PR TITLE
Add state estimation tutorial

### PR DESCRIPTION
**Pull request by Hustwireless**

### CODE REVIEW

File: docs/rss-tutorials/state_estimation_tutorial_preparations.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+# Preparations for State Estimation Tutorial
+For the tutorial you need the following packages:
+ - [ethzasl_msf](https://github.com/ethz-asl/ethzasl_msf)
+ - [compslam](https://bitbucket.org/leggedrobotics/compslam_rss/src/master/)
+
+You can test it on your own computer by running it with a rosbag provided [bellow](#Download files).
+
+## Installation
+Download the repositories and build them:
```
Comment (mantelt) [inline docs/rss-tutorials/state_estimation_tutorial_preparations.md]: Please extend these to account for the fact that the repositories needs to be added to the catkin workspace containing the smb_core environment .

File: docs/rss-tutorials/state_estimation_tutorial_preparations.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+```bash
+cd ~/catkin_ws/src/
+git clone https://github.com/ethz-asl/ethzasl_msf.git
+git clone https://github.com/ethz-asl/glog_catkin.git
+git clone https://github.com/catkin/catkin_simple.git
+catkin build loam
+catkin build ethzasl_msf
+catkin build smb_msf
+source ~/<catkin_ws>/devel/setup.bashrc
```
Comment (mantelt) [inline docs/rss-tutorials/state_estimation_tutorial_preparations.md]: please be consistent when specifying a path: use placeholders everywhere or nowhere.

**Review by mantelt — CHANGES_REQUESTED** [review]
Looks already ok to me. But please address my two minor comments.

File: docs/rss-tutorials/state_estimation_tutorial_preparations.md (hunk 1 above)
Comment (Hustwireless) [reply docs/rss-tutorials/state_estimation_tutorial_preparations.md]: Thanks, this should be specified.

File: docs/rss-tutorials/state_estimation_tutorial_preparations.md (hunk 2 above)
Comment (Hustwireless) [reply docs/rss-tutorials/state_estimation_tutorial_preparations.md]: Thanks for catching that!

**Pull request merged**